parallel-twitter-trending-hashtags/socket_receiver.py

```diff
@@ -0,0 +1,36 @@
+import socket
+import requests
+import ast
```

Comment:
Again, the ast thing. We're working with json, we should be using a json library.